db_stress_tool/expected_state.cc

```diff
@@ -67,20 +148,21 @@ int ExpectedState::DeleteRange(int cf, int64_t begin_key, int64_t end_key,
 }
 
 bool ExpectedState::Exists(int cf, int64_t key) {
-  // UNKNOWN_SENTINEL counts as exists. That assures a key for which overwrite
+  // Pending states count as "exist". That assures a key for which overwrite
   // is disallowed can't be accidentally added a second time, in which case
```

Comment:
[Note] can't really think of how the accident can happen ....

Reply:
It's again a crash recovery case. It could happen that (1) UNKNOWN_SENTINEL written as expected value, (2) write succeeds, (3) crash. Then upon recovery there's an UNKNOWN_SENTINEL for a key that exists in the DB.

Though I wonder if that's still the case after #9897...

Reply:
As discussed, this comment may no longer be true so I deleted it.